src/manager/kiwix-manage.cpp

```diff
@@ -96,24 +96,24 @@ void usage()
             << std::endl;
 }
 
-int handle_show(kiwix::Library* library, const std::string& libraryPath,
+int handle_show(kiwix::ConstLibraryPtr library, const std::string& libraryPath,
```

Comment:
I think that a parameter of type `const kiwix::Library&` is more suitable here (passing a `shared_ptr` suggests that it is preserved somewhere and can be accessed after the function returns).

Reply:
Agree.